src/test/java/com/michalsydoryk/app/board/Board2DTest.java

```diff
@@ -0,0 +1,159 @@
+package com.michalsydoryk.app.board;
+
+import com.michalsydoryk.app.coordinates.Coordinates2D;
+import com.michalsydoryk.app.sign.Sign;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+import java.util.ArrayDeque;
+import java.util.HashSet;
+import java.util.Queue;
+import java.util.Set;
+
+@Test
+public class Board2DTest {
+
+    public void shouldBeTrueIfCoordinatesAreInBord() {
+        //Given
+        int boardSize = 105;
+        Board2D board = new Board2D.Builder().boardSize(boardSize).build();
+        int x = 102;
+        int y = 2;
+        Coordinates2D coordinates = new Coordinates2D(x, y);
```

Comment:
This could be parametrized with dataProvider.

Reply:
I agree. will be fixed in future